src/sentry/api/endpoints/organization_search_details.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import (
+    OrganizationEndpoint,
+    OrganizationSearchPermission,
+)
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.models import SavedSearch
+
+
+class OrganizationSearchDetailsEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationSearchPermission, )
+
+    def delete(self, request, organization, search_id):
+        """
+        Delete a saved search
+
+        Permanently remove a saved search.
+
+            {method} {path}
+
+        """
+        try:
+            search = SavedSearch.objects.get(
+                owner__isnull=True,
```

Comment:
Are we going to add deleting pinned searches later? Or in separate endpoints?

Reply:
Yeah, separate pr incoming when I can figure out why github is rejecting it 🤷‍♂️